pkg/datamover/utils.go

```diff
@@ -87,8 +85,8 @@ func connectToKopiaRepositoryServer(ctx context.Context, rs *param.RepositorySer
 		hostname,
 		rs.Address,
 		rs.Username,
-		contentCacheMB,
-		metadataCacheMB,
+		rs.ContentCacheMB,
+		rs.MetadataCacheMB,
```

Comment:
Cachesettings fields are optional in the CR, hence we should check whether these fields are empty, if they are empty we should set the default cache settings - something like this https://github.com/kanisterio/kanister/pull/1918/files#diff-ff875fa901afef5392df922bc365afa2058035a9266f3c882b807d8b50440892R61.